os/manual-rollbacks.md

```diff
@@ -2,6 +2,8 @@
 
 As much as we all love flawless, automatic updates, there may be occasions when an update must be rolled back. This is fairly straightforward on Container Linux, once you know the right commands.
 
+**Note**: Container Linux allows you to rollback to a previously active partition, or change to an older release channel (to downgrade to a stable version). It does not allow you to downgrade to an uninstalled, user defined version number.
```

Comment:
I also half-suspect (not sure, suggesting here) that the location for this note might be _after_ the two rollback procedures, as a final clarifier.